domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -78,6 +78,8 @@
     private final YesNoOption paperResponse;
     private final LocalDateTime dateReferredForDirections;
     private final String preferredDQCourt;
+    private final ProceedOnPaperRequestType proceedOfflineReason;
+    private final String proceedOfflineOtherReason;
```

Comment:
No need for an optional getter for this one?

Reply:
For Enum type, I gave the optional getter to make easy in mapping from claimstore ENUM to CCD ENUM, but in case of the String, left it as its not populated, it goes as null.